src/internal/observable/dom/animationFrames.ts

```diff
@@ -77,20 +77,23 @@ export interface TimestampProvider {
  *
  * @param timestampProvider An object with a `now` method that provides a numeric timestamp
  */
-export function animationFrames(timestampProvider: TimestampProvider = Date) {
+export function animationFrames(timestampProvider?: TimestampProvider) {
   return timestampProvider === Date ? DEFAULT_ANIMATION_FRAMES : animationFramesFactory(timestampProvider);
```

Comment:
The default should be changed to **not** use `Date` as the timestamp provider, IMO. The purpose of the default is to avoid creating observable instances when `animationFrames` is called with the expected/default timestamp provider and the default is to use the timestamp provided by the API.

Reply:
Yes, I did forget to change the default here.